src/backend/mod.rs

```diff
@@ -109,8 +114,16 @@ pub trait Backend {
         }
     }
 
-    /// Get the size of the terminal screen as a [`Rect`].
-    fn size(&self) -> Result<Rect, io::Error>;
+    /// Get the size of the terminal screen in columns/rows as a [`Rect`].
+    fn size(&self) -> Result<Size, io::Error>;
```

Comment:
How about leaving this as Rect and just updating the window_size method?

otherwise nit: doc comment => Size instead of Rect